public/locales/en-US.json

```diff
@@ -205,13 +220,25 @@
     "header_Instance-hours": "Instance hours",
     "header_measurementType": "Type",
     "header_measurement_type": "Type",
+    "header_number_of_guests": "Guests",
     "header_inventoryId": "UUID",
+    "header_Sockets": "Sockets",
+    "header_Sockets_RHEL": "Capacity (sockets)",
+    "header_Sockets_RHEL for ARM": "$t(curiosity-inventory.header_Sockets_RHEL)",
```

Comment:
required: remove rhel archs, no longer needed with the filter PR #1014  that merged